include/mbedtls/ecp.h

```diff
@@ -1,10 +1,20 @@
 /**
  * \file ecp.h
  *
- * \brief Elliptic curves over GF(p)
+ * \brief This file contains ECP definitions and functions.
+ *
+ * The Elliptic Curve over P (ECP) is defined in <em>Standards for Efficient
```

Comment:
Please change to `The use of Elliptic Curves over GF(p) (ECP) in cryptography and TLS is defined in ...`.

Reply:
Accepted and updated